### PR TITLE
Migrate all deps to Conan Center Index variants

### LINKED ISSUE #10: Changes to naming convention Conan-Center
In recent developments, the conan center community has been changing the name convention for packages. These changes are being following by the bincrafters community.

An example is boost...
```sh
$ conan search -r conan-center boost
Existing package recipes:

boost/1.64.0@conan/stable
boost/1.65.1@conan/stable
boost/1.66.0@conan/stable
boost/1.67.0@conan/stable
boost/1.68.0@conan/stable
boost/1.69.0@conan/stable
boost/1.70.0
boost/1.70.0@conan/stable
boost/1.71.0
boost/1.71.0@conan/stable
```

They are no longer mentioning `@user/channel`.

Worse is that everything is no lower case... deps like `OpenSSL` became `openssl` which breaks the case sensitivity of conan.

Needless moving forward for future release the deps might need more complicate changes then just the version

Currently the OpenSSL is in conflict with some other packages that are now using openssl from bincrafters.

### PR DESCRIPTION
**Pull request by Croydon**
This migrates all dependencies to variants from the new [Conan Center Index](https://github.com/conan-io/conan-center-index)

Please also consider adding this recipe to the Index 😄 

Closes #10

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -22,13 +22,13 @@ class RestinioConan(ConanFile):
     build_policy = "missing"
 
     def requirements(self):
-        self.requires.add("http-parser/2.8.1@bincrafters/stable")
+        self.requires.add("http_parser/2.9.2")
```
Comment (prince-chrismc) [inline conanfile.py:25]: Is this a supported version change?

Comment (eao197) [reply conanfile.py:25]: We're using http-parser v.2.9.2 during the development of RESTinio for several month: https://github.com/Stiffstream/restinio/blob/master/externals.rb#L21

**Comment by eao197** [comment]
Thanks for PR. I'll use it for publishing version v.0.6.5 which is in the development now.

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
fix #10

**Pull request merged**